libmexclass/cpp/libmexclassInstall.cmake

```diff
@@ -0,0 +1,31 @@
+message(STATUS "Executing LIBMEXCLASS_INSTALL_STEPS_FILE: ${CMAKE_CURRENT_LIST_FILE} using INSTALL_COMMAND...")
+
+# ${CMAKE_CURRENT_LIST_DIR} points to the folder in which current executing script is located.
+# Current script LIBMEXCLASS_INSTALL_STEPS_FILE is in the libmexclass/cpp folder.
+set(LIBMEXCLASS_MATLAB_SOURCES "${CMAKE_CURRENT_LIST_DIR}/../matlab/+libmexclass/")
+message(STATUS "LIBMEXCLASS_MATLAB_SOURCES: ${LIBMEXCLASS_MATLAB_SOURCES}")
+
+# Copy ${LIBMEXCLASS_MATLAB_SOURCES} first so that ${CMAKE_CURRENT_BINARY_DIR}/+libmexclass/+proxy/ folder is there to which we copy necessary binaries later.
+set(CMAKE_PACKAGED_LIBMEXCLASS_INSTALL_DIR "${CMAKE_CURRENT_BINARY_DIR}/+libmexclass")
+message(STATUS "CMAKE_PACKAGED_LIBMEXCLASS_INSTALL_DIR: ${CMAKE_PACKAGED_LIBMEXCLASS_INSTALL_DIR}")
+message(STATUS "Copying LIBMEXCLASS_MATLAB_SOURCES to packaged folder CMAKE_PACKAGED_LIBMEXCLASS_INSTALL_DIR...")
+file(COPY ${LIBMEXCLASS_MATLAB_SOURCES} DESTINATION ${CMAKE_PACKAGED_LIBMEXCLASS_INSTALL_DIR})
+
+set(ACTUAL_BINARY_DIR ${CMAKE_CURRENT_BINARY_DIR})
+if(WIN32)
+	# On Windows, the Visual Studio build adds an additional "Release" folder to place the output binaries.
+	set(ACTUAL_BINARY_DIR "${ACTUAL_BINARY_DIR}/Release/")
+	list(APPEND INSTALL_BINARIES_LIST ${ACTUAL_BINARY_DIR}/gateway.mexw64 ${ACTUAL_BINARY_DIR}/mexclass.dll)	
```

Comment:
I think we also need to copy over any shared libraries that the client `proxy::Factory` depends on (i.e. `${CUSTOM_PROXY_LINK_LIBRARIES}` to the same directory as the MEX gateway and `libmexclass.dll`

Reply:
Resolved in commit 72a2e7f. See the last `install` steps in the diff for [`libmexclass/cpp/CMakeLists.txt`](https://github.com/mathworks/libmexclass/commit/72a2e7f14cc43fd75dcb298f9841f58238b5db03#diff-7d757216dff7d3606c1796c8225fec917ec2704dd09a8bbc1448f372255e3d90).